app/client/src/features/dashboard/Dashboard.tsx

```diff
@@ -60,63 +42,59 @@ export const DASHBOARD_QUERY = graphql`
-            }),
-        [listOfEvents]
-    );
+    const refresh = React.useCallback(() => {
+        if (isRefreshing) return;
+        setIsRefreshing(true);
+        fetchQuery(env, DASHBOARD_QUERY, {}).subscribe({
+            complete: () => {
+                setIsRefreshing(false);
+                loadQuery({}, { fetchPolicy: 'store-or-network' });
+            },
+            error: () => {
+                setIsRefreshing(false);
+            },
+        });
+    }, [env, isRefreshing, loadQuery]);
 
-    // Store upcoming events
-    const upcomingEvents = React.useMemo(
-        () =>
-            listOfEvents.filter(({ node: event }) => {
-                if (!event.startDateTime) return false;
-                const now = new Date();
-                return isAfter(new Date(event.startDateTime), now);
-            }),
-        [listOfEvents]
-    );
+    React.useEffect(() => {
+        if (!queryRef) loadQuery({});
+    }, [queryRef, loadQuery]);
 
-    if (!data) return <Loader />;
+    React.useEffect(() => {
+        const interval = setInterval(() => {
+            refresh();
+        }, REFRESH_INTERVAL);
+        return () => clearInterval(interval);
+    });
 
     return (
-        <Grid container>
-            <DashboardEventList eventList={ongoingEvents} ongoing={true} />
-            <DashboardEventList eventList={upcomingEvents} ongoing={false} />
-            <Grid item>
-                <Card>
-                    <CardContent style={{ display: 'flex', justifyContent: 'center', padding: 12 }}>
-                        <IconButton aria-label='view future event' onClick={handleNav('/organizations/me')}>
-                            <Add className={classes.addIcon} />
-                        </IconButton>
-                    </CardContent>
-                </Card>
-                <Typography variant='subtitle2' className={classes.addIconTitle}>
-                    Create Event
-                </Typography>
-            </Grid>
-        </Grid>
+        <ConditionalRender client>
+            <React.Suspense fallback={<div>Loading...</div>}>
+                <Grid container>
+                    {queryRef ? <DashboardEventList queryRef={queryRef} /> : <></>}
+                    <Grid item>
+                        <Card>
+                            <CardContent style={{ display: 'flex', justifyContent: 'center', padding: 12 }}>
+                                <IconButton aria-label='view future event' onClick={handleNav('/organizations/me')}>
+                                    <Add className={classes.addIcon} />
+                                </IconButton>
+                            </CardContent>
+                        </Card>
+                        <Typography variant='subtitle2' className={classes.addIconTitle}>
+                            Create Event
+                        </Typography>
+                    </Grid>
```

Comment:
This card grid item can be removed, organizers will create events through the Org tab and since they can have multiple organizations trying to create an event through the dashboard would complicate things.